lib/base/networking/protocols/grpc/campus.pb.dart

```diff
@@ -0,0 +1,5980 @@
+//
```

Comment:
is this for grpc in general or for the campus backend? if it's for the latter I would move the folder to apis/campusBackend

Reply:
Depends: if we add further `.proto` files in the future, they will be added the the `protos` directory too => adding this to a "neutral" directory somewhat makes sense
(at least all guides on the internet I watched did it this way => I assumed this is somewhat best practice)

Reply:
Does that happen automatically?

Reply:
Does what happen automatically?

Reply:
I have added them to the api folder